File: AtividadeExtra02/Classes/ControleDeAudio.java

public class ControleDeAudio {

    int volume;

    public ControleDeAudio() {
        this.volume = 2;
    }

    public void aumentarVolume() {
        if (volume == 10) {
            System.out.println("\nVolume já está no máximo! Volume: " + volume);
            return;
        }
        this.volume++;
    }

    public void diminuirVolume() {
        if (volume == 1) {
            System.out.println("\nVolume já está no mínimo! Volume: " + volume);
            return;
        }
        this.volume--;
    }

    public int lerVolume() {
        return this.volume;
    }

}

File: AtividadeExtra02/Classes/ControleDeAudio.ts
class ControleDeAudio {
    volume : number;

    constructor() {
        this.volume = 2;  
    }

    aumentarVolume() : void {
        if (this.volume == 10){
            console.log("\nVolume já está no máximo! Volume: " + this.volume);
            return;
        }
        this.volume += 1;
    }

    diminuirVolume() : void {
        if (this.volume == 1){
            console.log("\nVolume já está no mínimo! Volume: " + this.volume);
            return;
        }
        this.volume -= 1;
    }

    lerVolume() : number {
        return this.volume;
    }

}

function main(){
    const c : ControleDeAudio = new ControleDeAudio();

    c.aumentarVolume();
    c.aumentarVolume();
    c.diminuirVolume();
    c.aumentarVolume();
    c.diminuirVolume();

    console.log("Volume = " + c.lerVolume());  // Saída: Volume = 3
}

main();